src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -312,6 +308,55 @@ void testCreateElectionsForDarCollection() throws Exception {
     verify(darCollectionDAO, times(1)).findDARCollectionByCollectionId(any());
   }
 
+  @Test
+  void testCreateElectionsForDarCollectionEmpty() {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+
+    assertThrows(IllegalStateException.class, () -> {
+      service.createElectionsForDarCollection(user, collection);
+    });
+  }
+
+  @Test
+  void testCreateElectionsForDarCollectionNull() throws Exception {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(null);
+
+    assertThrows(IllegalStateException.class, () -> {
+      service.createElectionsForDarCollection(user, collection);
+    });
+  }
+
+  @Test
+  void testCreateElectionsForDarCollectionVoteUsersException() throws Exception {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(
+        List.of("electionId"));
+    when(voteDAO.findVoteUsersByElectionReferenceIdList(any())).thenThrow(
+        IllegalArgumentException.class);
+
+    service.createElectionsForDarCollection(user, collection);
+    verify(darCollectionServiceDAO, times(1)).createElectionsForDarCollection(any(), any());
```

Comment:
`times(1)` is the default so you can omit it
```suggestion
    verify(darCollectionServiceDAO).createElectionsForDarCollection(any(), any());
```

Also I would encourage matching against more specific arguments than `any()` if possible. You can use `ArgumentCaptor` to assert on a subfield of an argument in cases where you can't match the entire argument.